Bonsai.Templates/Bonsai.Templates/source.extension.vsixmanifest

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" ?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
-    <Identity Id="BonsaiTemplatesVsix.ac37e8b9-5d6c-407c-b2ca-2e550a8ab918" Version="2.8.2" Language="en-US" Publisher="Bonsai Foundation" />
+    <Identity Id="BonsaiTemplatesVsix.ac37e8b9-5d6c-407c-b2ca-2e550a8ab918" Version="|%CurrentProject%;GetVsixVersion|" Language="en-US" Publisher="Bonsai Foundation" />
```

Comment:
Didn't know you could do this, not familiar with this syntax, that's interesting.

Reply:
Yup! I didn't know about it until this PR either. It basically invokes this MSBuild target to get the version it should use:

https://github.com/bonsai-rx/bonsai/blob/93503f2a059eda7381ed20d74d69fb0923a8ae02/Bonsai.Templates/Bonsai.Templates/Bonsai.Templates.csproj#L106-L108

I discovered it by looking at the Roslyn source code ([example](https://github.com/dotnet/roslyn/blob/967907da1e4846c15b30a68e37ddcdfc9964781a/src/Compilers/Extension/source.extension.vsixmanifest#L4)) since I knew they deploy a lot of VSIX and figured there was no way they were manually bumping versions.

During our meeting I incorrectly stated that this target is normally something in the modern .NET SDK. It actually turns out it's [part of .NET Arcade](https://github.com/dotnet/arcade/blob/4a7d983f833d6b86365ea1b2b4d6ee72fbdbf944/src/Microsoft.DotNet.Arcade.Sdk/tools/VisualStudio.targets#L31-L32) This made me realize that the SDK-style VSIX projects used by Roslyn probably come from there too, which would definitely explain why I couldn't find them documented anywhere 😅